config-coral.ini

```diff
@@ -43,6 +43,8 @@ Id = default
 Emails =
 NotifyEveryMinutes = 0
 ViolationThreshold = 60
+#wether repeat the video processing when video reached to its final frame or not.
+Repeat = false
```

Comment:
```suggestion
# Wether to loop the video processing when video reached to its final frame or not.
Repeat = false
```
This config allows for starting all over the video once it's finished?  Could we change the name of the config to something like `LoopVideoFile`. Something that's clear that affects only when consuming a video file after it finishes. And not a camera

Reply:
```suggestion
# Whether repeat the video processing when video reached to its final frame or not.
Repeat = false
```

Reply:
Yeah, you are right. Done